### DISCUSSION TITLE
Disable cache webserver

### DISCUSSION THREAD

**Discussion by centratelemedia** [activej/activej#230] [General]
my be any option to disable cache in development mode, i try to disable in html file but its not work, 

```
<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
<meta http-equiv="Pragma" content="no-cache" />
<meta http-equiv="Expires" content="0" />
```

**Comment by eduard-vasinskyi** [comment]
Hi, @centratelemedia

ActiveJ does not enable caching implicitly. If you have explicitly enabled caching for a web page, you can also programmatically control whether or not to enable caching by some condition.

Also, a common approach is to use the browser’s Development Mode (usually enabled by pressing the F12 key), where you can explicitly clear the cache on reload.
